src/backend/db/quizCategory.js

```diff
@@ -0,0 +1,32 @@
+import { v4 as uuid } from "uuid";
+
+const quizeCategories = {
```

Comment:
suggestion: I think `quizCategories` got misspelled

Reply:
Done, in the upcoming PR